### PR TITLE
[visrtx] New port

### PR DESCRIPTION
**Pull request by acdemiralp**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] The name of the port matches an existing name for this component on https://repology.org/ if possible, and/or is strongly associated with that component on search engines.
- [x] Optional dependencies are resolved in exactly one way. For example, if the component is built with CMake, all `find_package` calls are REQUIRED, are satisfied by `vcpkg.json`'s declared dependencies, or disabled with [CMAKE_DISABLE_FIND_PACKAGE_Xxx](https://cmake.org/cmake/help/latest/variable/CMAKE_DISABLE_FIND_PACKAGE_PackageName.html)
- [x] The versioning scheme in `vcpkg.json` matches what upstream says.
- [x] The license declaration in `vcpkg.json` matches what upstream says.
- [x] The installed as the "copyright" file matches what upstream says.
- [x] The source code of the component installed comes from an authoritative source.
- [x] The generated "usage text" is accurate. See [adding-usage](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/examples/adding-usage.md) for context.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is in the new port's versions file.
- [x] Only one version is added to each modified port's versions file.

Notes to the reviewer: This is opened as a draft until https://github.com/microsoft/vcpkg/pull/33304 is merged as it depends on OptiX.

### CODE REVIEW

File: ports/visrtx/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "version": "0.6.1",
+  "description": "Experimental, scientific visualization-focused implementation of the Khronos ANARI standard.",
+  "homepage": "https://github.com/NVIDIA/VisRTX",
+  "license": "BSD-3-Clause",
+  "dependencies": [
+    "anari",
+    "cuda",
+    "glm",
+    "opengl",
+    "optix",
```
Comment (JonLiu1993) [inline ports/visrtx/vcpkg.json:12]:
```suggestion
```

Comment (JonLiu1993) [reply ports/visrtx/vcpkg.json:12]: Vcpkg does not have an `optix` port.

Comment (acdemiralp) [reply ports/visrtx/vcpkg.json:12]: It hopefully will. See #33304 . This MR will be marked ready for review once #33304 is resolved.

**Comment by acdemiralp** [comment]
See #33304 .

**Pull request closed without merging**